content/docs/3.0/tasks/deploy/docker-compose.md

```diff
@@ -1,165 +1,20 @@
-      - --log-file=/logs/tikv0.log
-    depends_on:
-      - "pd0"
-    restart: on-failure
-```
-
-All the following example docker compose config file will contain this base config.
-
-## Example
-
-### Run [GO-YCSB](https://github.com/pingcap/go-ycsb) to connect to the TiKV cluster:
-
-1. Create a `ycsb-docker-compose.yml` file, add the above base config to this file, and then append the following section:
-
-    ```yaml
-    ycsb:
-      image: pingcap/go-ycsb
-    ```
-
-2. Start the cluster:
-
-    ```bash
-    rm -rf data logs 
-    docker-compose -f ycsb-docker-compose.yml pull 
-    docker-compose -f ycsb-docker-compose.yml up -d
-    ```
-
-3. Start YCSB:
-
-    ```bash
-    docker-compose -f ycsb-docker-compose.yml run ycsb shell tikv -p tikv.pd=pd0:2379
-    ```
-
-4. Run YCSB:
-
-    ```bash
-    INFO[0000] [pd] create pd client with endpoints [pd0:2379]
-    INFO[0000] [pd] leader switches to: http://pd0:2379, previous:
-    INFO[0000] [pd] init cluster id 6628733331417096653
-    » read a
-    Read empty for a
-    » insert a field0=0
-    Insert a ok
-    » read a
-    Read a ok
-    field0="0"
-    ```
-
-### Use [Titan](https://github.com/meitu/titan) to connect to the TiKV cluster via the Redis protocol:
-
-1. Create a `titan-docker-compose.yml` file, add the above base config to this file, and then append the following section:
-
-    ```yaml
-      titan:
-        image: meitu/titan
-        ports:
-          - "7369:7369"
-        command:
-          - --pd-addrs=tikv://pd0:2379
-        depends_on:
-          - "tikv0"
-        restart: on-failure    
-    ```
-
-2. Start the cluster:
-
-    ```bash
-    rm -rf data logs 
-    docker-compose -f titan-docker-compose.yml pull
-    docker-compose -f titan-docker-compose.yml up -d
-    ```
-
-3. Use `redis-cli` to communicate with Titan:
-
-    ```bash
-    redis-cli -p 7369
-    127.0.0.1:7369> set a 1
-    OK
-    127.0.0.1:7369> get a
-    "1"
-    ```
-
-## What's next?
-
-+ If you want to try the Go client, see [Try Two Types of APIs](../clients/go-client-api.md). You need to build your docker image and add it to the docker compose config file like above YCSB or Titan does. 
-+ If you want to run a full cluster with monitor support, please follow the [tidb-docker-compose guide](https://github.com/pingcap/tidb-docker-compose/blob/master/README.md), comment the `tidb` and `tispark` sections out in the [values.yaml](https://github.com/pingcap/tidb-docker-compose/blob/master/compose/values.yaml), generate the new docker compose config, then add your own binary image and run it.
+We recommend using the [**Docker Swarm option**](https://github.com/pingcap/tidb-docker-compose#docker-swarm) option for using TiKV. The [Docker Compose](https://github.com/pingcap/tidb-docker-compose#quick-start) option requires some [manual configuration to interact with TiKV directly](https://github.com/pingcap/tidb-docker-compose#host-network-mode-linux) on non-Linux systems.
```

Comment:
```suggestion
For non-Linux systems, the [Docker Compose](https://github.com/pingcap/tidb-docker-compose#quick-start) option requires some [manual configuration to interact with TiKV directly](https://github.com/pingcap/tidb-docker-compose#host-network-mode-linux). In this case, we recommend using the [**Docker Swarm**](https://github.com/pingcap/tidb-docker-compose#docker-swarm) option for deploying TiKV.
```

Reply:
Adjusted the sequence to make the recommendation sound more convincing

Reply:
Hmm, I'd prefer to recommend Swarm mode in all cases as I don't really want to have to support the Docker Compose method. :( It's technology that is on the way out (Swarm is replacing it) and has a lot of dependencies.

Reply:
I tried to clean this up.